src/components/Card.jsx

```diff
@@ -1,26 +1,52 @@
+import { useState } from 'react';
+
 import { calculateDateDifference, formatDate } from '../utils/dateCalculate';
+
 import defaultImg from '../assets/logo.svg';
 import starIcon from '../assets/icon/star.svg';
 import kebabIcon from '../assets/icon/kebab.svg';
 
 import './Card.css'
+import ModalLinkAdd from './modals/ModalLinkAdd';
+
+const Card = ({imgSrc, createdAt, title, description, url, folderName, folderList}) => {
+  const [isOpenKebab, setIsOpenKebab] = useState(false);
+  const [isOpenModalLinkAdd, setIsOpenModalLinkAdd] = useState(false);
+
+  const handleClickKebab = () => {
+    setIsOpenKebab(!isOpenKebab);
+  }
+
+  const handleClickModalLinkAdd = () => {
+    setIsOpenModalLinkAdd(!isOpenModalLinkAdd);
+  }
 
-const Card = ({imgSrc, createdAt, title, description, url}) => {
   return (
+    <>
     <li className='card'>
       <a href={url} target="_blank" rel="noreferrer noopener">
         <img src={imgSrc ?? defaultImg} alt='이미지' className='card-img' />
       </a>
         <div className='card-info'>
           <div className='container-kebab'>
             <span className='time'>{calculateDateDifference(createdAt)}</span>
-            <img src={kebabIcon} alt='목록' className='card-info_kebab' />
+            <img src={kebabIcon} alt='목록' className='card-info_kebab' onClick={handleClickKebab}/>
+            {isOpenKebab && (
+              <div>
+                <button>삭제 하기</button>
+                <button onClick={handleClickModalLinkAdd}>폴더에 추가</button>
```

Comment:
type attribute를 의식적으로 신경써주시면 좋을 것 같아요~
```suggestion
                <button type='button'>삭제 하기</button>
                <button type='button' onClick={handleClickModalLinkAdd}>폴더에 추가</button>
```